packages/typespec-vscode/src/utils.ts

```diff
@@ -192,14 +192,19 @@ export function spawnExecutionAndLogToOutput(
   exe: string,
   args: string[],
   cwd: string,
+  logStderrAsError: boolean,
   env?: NodeJS.ProcessEnv,
 ): Promise<ExecOutput> {
   return spawnExecution(exe, args, cwd, env, {
```

Comment:
why don't you collect the outpu then and just log one single info or error depending on the status code with the result if its so important to log errors differently. That would be the most accurate without any special casing